Client/Frontend/Home/HomepageViewModel.swift

```diff
@@ -218,7 +220,8 @@ class HomepageViewModel: FeatureFlaggable {
         ]
          */
         // Ecosia: Those models needs to follow strictly the order defined in `enum HomepageSectionType`
-        self.childViewModels = [headerViewModel,
+        self.childViewModels = [climateImpactCounterViewModel,
+								headerViewModel,
                                 onboardingCardViewModel,
```

Comment:
Looks like indentation got a bit wrong here? Or maybe it's github tricking me 😅 

```suggestion
        self.childViewModels = [climateImpactCounterViewModel,
                                headerViewModel,
                                onboardingCardViewModel,
```